ui/src/main/resources/PhenoTips/MatchContact.xml

```diff
@@ -54,7 +54,10 @@
       'groups'   : "#if($options.groups)${groups}#end",
 
       'subject'  : "[$!{platformName}] $!{options.subject}",
-      'message'  : "$!{options.message}"
+      'message'  : "$!{options.message}",
+
+      'patientURL': $xwiki.getDocument($services.patients.getPatientById("$!{request.patient}").getDocument()).getExternalURL(),
+      'matchURL'  : $xwiki.getDocument($services.patients.getPatientById("$!{request.matchId}").getDocument()).getExternalURL()
```

Comment:
Yes, I was not sure this is what we want to do. It's going to be viewed by a user who surely has a right to view the patient, but it might mislead the user who sends the email.